cookietemple/lint/template_linter.py

```diff
@@ -583,3 +593,103 @@ def init_changelog_lint_content(self) -> list:
+            check_section_flag &= self.check_section(self.parser.items('sync_files_blacklisted'), 'sync_files_blacklisted', self.linter_ctx)
+            if check_section_flag:
+                self.linter_ctx.passed.append(('general-7', 'All config sections passed cookietemple linting!'))
+        else:
+            self.linter_ctx.failed.append(('general-7', 'Aborted config section linting. Fix missing sections first!'))
+
+    def check_missing_sections(self, parsed_sections) -> bool:
+        """
+        Examine cookietemple config file for missing sections
+        :param parsed_sections: All parsed sections from the user's cookietemple.cfg file
+        """
+        sections = ['bumpversion', 'bumpversion_files_whitelisted', 'bumpversion_files_blacklisted', 'sync_files_blacklisted', 'sync_level']
+        missing_sections = []
+        # for every section check whether it is in the parsed sections or not
+        for section in sections:
+            if section not in parsed_sections:
+                missing_sections.append(section)
+        # if there were any missing sections, let linter fail
+        if missing_sections:
+            miss_section_info = 'Cookietemple config file misses section' + 's' if len(missing_sections) > 1 else ''
+            self.linter_ctx.failed.append(('general-7', f'{miss_section_info}: {" ".join(section for section in missing_sections)}'))
+            return False
+        else:
+            self.linter_ctx.passed.append(('general-7', 'All required cookietemple.cfg sections were found!'))
+            return True
+
+    def check_section(self, section_items, section_name: str, main_linter: TemplateLinter) -> bool:
+        """
+        Check requirements 2.) - 5.) stated above.
+        :param section_items: A pair (name, value) for all items in a section
+        :param section_name: The sections name
+        :param main_linter: The linter calling the function
+        """
+        # a set containig the section name as a key and its linting rules as values
+        # linting rules made of:
+        #   1.) a tuple with a variable_name and its value ('*' if value does not care for linting)
+        #   => NOTE: if one variable can have multiple valid values, they are separated by '|'
+        #
+        #   2.) a number, stating the minimum number of items in a section (-1 if number does not care)
+        check_set = {
+            'bumpversion': [('current_version', '*'), 1],
+            'bumpversion_files_whitelisted': [('dot_cookietemple', '.cookietemple.yml'), -1],
+            'sync_level': [('ct_sync_level', 'patch|minor|major'), 1],
+            'sync_files_blacklisted': [('changelog', 'CHANGELOG.rst'), -1]
+        }
+        return ConfigLinter.apply_section_linting_rules(section_name, section_items, check_set.get(section_name), main_linter)
+
+    @staticmethod
+    def apply_section_linting_rules(section: str, section_items, section_lint_rules, main_linter: TemplateLinter) -> bool:
```

Comment:
Not sure whether "apply" really is the correct word here.

Apply always sounds like doing something which is supposed to last.

We're just checking something here.